OnlineShopUI/online-shop-ui/src/components/Header.scss

```diff
@@ -0,0 +1,19 @@
+.header {
+  &__contact {
+    display: flex;
+    justify-content: space-between;
+    padding: 16px 45px 16px 45px;
+    > ul {
+      display: flex;
+      gap: 30px;
+      font-family: 'Open Sans', sans-serif;
+      font-size: 12px;
+    }
+  }
+  > hr {
+    width: 95%;
+    margin: 0 auto;
+    background-color: #d1d1d1;
+    border-color: #d1d1d1;
```

Comment:
Colours hard-coded, it will be difficult to change colour palette of the site if we provide custom colour theme of the site